CONTRIBUTING.md

```diff
@@ -374,9 +374,9 @@ of the **Concept Approval Committee**.
 
 **Committee Members**
 
-* Core Members: **Aaron** (Regen), **Bez** (Fission), **Alessio** (AiB)
```

Comment:
We need a chair too.

Reply:
What will be the role of chair?

Reply:
Chairs have an important role in governance systems in which _members_ vote as per _clear_ voting mechanisms.
For example, see https://www.debian.org/devel/constitution. Chairs typically gather bullet points for meeting agendas and establish the order in which the subjects are to be discussed. They are _the guardians of processes and procedures_. They timebox and moderate conversations. Chains are either elected or appointed and typically don't vote if they are elected among the members of the board. In certain _well-defined_ circumstances they may be granted a casting vote.